apps/judicial-system/web/src/components/IndictmentCaseFilesList/IndictmentCaseFilesList.tsx

```diff
@@ -210,34 +210,29 @@ const IndictmentCaseFilesList: React.FC<React.PropsWithChildren<Props>> = (
           </Box>
         ))}
       </Box>
-      {isDistrictCourtUser(user) || isCompletedCase(workingCase.state) ? (
-        <>
+      {(isDistrictCourtUser(user) || isCompletedCase(workingCase.state)) &&
+      (courtRecords?.length || rulings?.length) ? (
```

Comment:
Please consider using `courRecords?.length > 0 || rulings?.length > 0` for clarity.

Reply:
I was going to do that at first, but because the courtRecords and rulings can be undefined I'd have to do (courtRecords?.length ?? 0) > 0 which I feel is even less clear, and then we check the length a little further down so it also felt redundant. We could also default the CaseFile arrays here to being empty instead of undefined I guess but I think it's fine like this for now.

Reply:
Agree